views/login.hbs

```diff
@@ -32,26 +32,36 @@
     <div class='wrapper'>
         <!-- NAVBAR -->
         {{> navbar }}
-
-        <div class="houseline"></div>
+        {{!-- <div class="houseline"></div> --}}
 
         <div class="background theme-yellow col-lg-6 col-lg-offset-3 col-md-8 col-md-offset-2 col-xs-10 col-xs-offset-1">
```

Comment:
Another small thing, both for this page and the registration page, but you can remove `.theme-yellow` from these since the class doesn't exist anymore (and is a bit misleading now)